templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -402,13 +355,17 @@ Parameters:
       - db.t2.large
       - db.t2.xlarge
       - db.t2.2xlarge
+      - db.t3.medium
+      - db.t3.large
+      - db.t3.xlarge
+      - db.t3.2xlarge
```

Comment:
If we're adding the `t3` family, could we add the `r5` family of databases as well?
```
      - db.r5.large
      - db.r5.xlarge
      - db.r5.2xlarge
      - db.r5.4xlarge
      - db.r5.12xlarge
      - db.r5.24xlarge
```

Reply:
The reason for adding the t3 family is because it is already present on the other Bitbucket DC template (for deploying into an existing ASI). I could add the r5 family as well

Reply:
That would be great thank you 😄

Reply:
✅